features/step_definitions/flash_steps.rb

```diff
@@ -1,3 +1,7 @@
 Then /^I should see the flash message "([^"]*)"$/ do |flash_text|
   find(:xpath, "//div[@id='flash-messages']").text.should have_content(flash_text)
 end
+
+Then /^I should see a flash message "([^"]*)"$/ do |flash_text|
+  find(:xpath, "//div[@id='flashWrapper']").text.should have_content(flash_text)
```

Comment:
Maybe there's a way to find in multiple `:xpath` ? then I could just use the original step? cc: @hallelujah @guicassolato @Martouta ?

Reply:
This is what you're looking for:
```
find(:xpath, "//div[@id='flash-messages'] | //div[@id='flashWrapper']").text.should have_content(flash_text)
```